packages/react-scripts/scripts/eject.js

```diff
@@ -38,6 +38,20 @@ function getGitStatus() {
   }
 }
 
+function tryGitAdd(appPath) {
+  try {
+    spawnSync(
+      'git',
+      ['add', path.join(appPath, 'config'), path.join(appPath, 'scripts')],
+      {
+        stdio: 'inherit',
+      }
+    );
+  } catch (e) {
+    return '';
```

Comment:
You can probably just `return` instead of returning an empty string. What get's returned if this succeeds?

Reply:
I thought about just returning but then I saw we're retuning an empty string  on line [#37](https://github.com/facebook/create-react-app/pull/5960/files#diff-64d71298908161d0a835f0b72170c307R37) so I did to keep it consistent. 

I forgot to return `true` so I'll add that in. 😓

Reply:
That other function returns a string on success and an empty string on failure. If this function returns `true` on success then it should return `false` on failure.